functions-and-operators/json-functions.md

```diff
@@ -35,18 +35,16 @@ TiDB supports most of the JSON functions that shipped with the GA release of MyS
 | Function Name                     | Description |
 | --------------------------------- | ----------- |
 | [JSON_APPEND(json_doc, path, value)](https://dev.mysql.com/doc/refman/8.0/en/json-modification-functions.html#function_json-append) | An alias to `JSON_ARRAY_APPEND` |
-| [JSON_ARRAY_APPEND(json_doc, path, value)](https://dev.mysql.com/doc/refman/8.0/en/json-modification-functions.html#function_json-array-append) | Appends a value to the end of a JSON array at a specified path |
-| [JSON_ARRAY_INSERT(json_doc, path, val[, path, val] ...)](https://dev.mysql.com/doc/refman/8.0/en/json-modification-functions.html#function_json-array-insert) | Inserts an array into the json document and returns the modified document |
+| [JSON_ARRAY_APPEND(json_doc, path, val[, path, val] ...)](https://dev.mysql.com/doc/refman/8.0/en/json-modification-functions.html#function_json-array-append) | Appends values to the end of the indicated arrays within a JSON document and returns the result |
+| [JSON_ARRAY_INSERT(json_doc, path, val[, path, val] ...)](https://dev.mysql.com/doc/refman/8.0/en/json-modification-functions.html#function_json-array-insert) | Insert values into the specified location of a JSON document and returns the result |
```

Comment:
Maybe be consistent with singular or plural of `arrays` vs `location`, since both seems to take one or more paths?
I.e. either have both `indicated array` and `specified location` *or* `indicated arrays` and `specified locations`?

Reply:
@qiancai I also noticed `JSON document` vs `json document` here. Which one should it be?

Reply:
`JSON document` is more proper as `JSON` is the correct format name  (see https://www.json.org/json-en.html)